### PR TITLE
feat: add Threads for cgroupv2

### PR DESCRIPTION
**Pull request by Rouzip**
Add `Threads` for cgroup v2 like `Tasks(Name, bool) ([]Task, error)` in cgroup v1.

### CODE REVIEW

**Comment by Rouzip** [comment]
PTAL, thanks. @AkihiroSuda

File: cgroup2/manager.go [hunk 1]
```diff
@@ -504,6 +504,32 @@ func (c *Manager) Procs(recursive bool) ([]uint64, error) {
...
+		if name != cgroupThreads {
+			return nil
+		}
+		procs, err := parseCgroupProcsFile(p)
+		if err != nil {
+			return err
+		}
+		threads = append(threads, procs...)
+		return nil
+	})
```
Comment (dcantah) [inline cgroup2/manager.go:529]: Given this is shared between `Procs` and this, can we split this into a helper that just takes in the name to care about? e.g. pass in cgroupThreads for this and cgroupProcs for `Procs`

Comment (Rouzip) [reply cgroup2/manager.go:529]: Hello, if you mean something like [this](https://github.com/Rouzip/runc/blob/50e7b9cb9605aa3409f96267aa341b15f24f862d/libcontainer/cgroups/utils.go#L140-L162)?

**Comment by Rouzip** [comment]
Hello, any advices are welcome. PTAL @dcantah

**Review by dcantah — APPROVED** [review]

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**